test-specifications/cryptoService/accountCreateTransaction.md

```diff
@@ -0,0 +1,155 @@
+| declineStakingReward      | bool   | optional          |                                                                                                                                             |
+| alias                     | string | optional          | Hex string representation of a serialized protobuf Key of an ED25519 or ECDSAsecp256k1 public key type.                                     |
+
+## Property Tests
+
+### **Key:**
+
+- The key for the new account.
+
+| Test no | Name                                                                                          | Input                                   | Expected response                                                              | Implemented (Y/N) |
+|---------|-----------------------------------------------------------------------------------------------|-----------------------------------------|--------------------------------------------------------------------------------|-------------------|
+| 1       | Creates an account with a valid ED25519 public key                                            | key=<VALID_ED25519_PUBLIC_KEY>          | The account creation succeeds.                                                 | N                 |
+| 2       | Creates an account with a valid ECDSAsecp256k1 public key                                     | key=<VALID_ECDSA_SECP256K1_PUBLIC_KEY>  | The account creation succeeds.                                                 | N                 |
+| 3       | Creates an account with a valid ED25519 private key                                           | key=<VALID_ED25519_PRIVATE_KEY>         | The account creation succeeds.                                                 | N                 |
+| 4       | Creates an account with a valid ECDSAsecp256k1 private key                                    | key=<VALID_ECDSA_SECP256K1_PRIVATE_KEY> | The account creation succeeds.                                                 | N                 |
+| 5       | Creates an account with a valid KeyList of ED25519 and ECDSAsecp256k1 private and public keys | key=<VALID_KEYLIST>                     | The account creation succeeds.                                                 | N                 |
+| 6       | Creates an account with a valid KeyList of nested Keylists (three levels)                     | key=<VALID_NESTED_KEYLIST>              | The account creation succeeds.                                                 | N                 |
+| 7       | Creates an account with no key                                                                |                                         | The account creation fails with a KEY_REQUIRED response code from the network. | N                 |
+| 8       | Creates an account with an invalid key                                                        | key=<INVALID_KEY>                       | The account creation fails with an SDK internal error.                         | N                 |
```

Comment:
I debated adding what the JSON will look like for each test, but thought that would be too cumbersome.. maybe just one example for each property/function? Open to thoughts here

Reply:
LGTM